app/forms/common/YesNoAnswer.scala

```diff
@@ -44,3 +47,13 @@ object YesNoAnswer {
   def form(fieldName: String = "yesNo", errorKey: String = "error.yesNo.required"): Form[YesNoAnswer] =
     Form(mapping(fieldName, errorKey))
 }
+
+object IsAuthorisationRequired extends ClearanceModifierPage {
+  override def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] =
+    Seq(TariffContentKey(s"tariff.declaration.isAuthorisationRequired.${defineJourneySpecificKey(decType)}"))
+}
+
+object PreviousDocumentsRemove extends DeclarationPage {
```

Comment:
I am wondering, why any 'remove' page would require tariff to be present?

Reply:
This is in keeping with previous behavior.